### DISCUSSION TITLE
Can I import a Secure CRT config?

### DISCUSSION THREAD

**Discussion by Exosen** [Eugeny/tabby#5881] [General]
I just want to know if I can use an import from Secure CRT. I keep getting annoying stuff on Secure CRT. And found Tabby which I like. But I have like over 200+ SSH sessions. And I am not willing to do this manually 1 by 1.

Thanks.

**Comment by wl4g** [comment]
I have encountered the same problem. How did you solve it？

**Comment by robmarzialetti** [comment]
I have the same issue.

This is my workaround:

`app-root { font-size: 1.25em; }`

in Appearence -> CustomCSS
